### ISSUE TITLE
JSON field does not work anymore in MySQL 5.7+

### ISSUE THREAD

**Issue by ksfreitas** [nielsgl/sequelize-paper-trail#91]
JSON field was changed to JSONB (I think its a Postgres specific field). In past versions of paper-trail, the JSON field it worked fine with MySQL 5.7+ versions.

I'm working in a patch to keep paper trail working both with Pg/MySQL native JSON fields.

**Comment by EdByrnee** [comment]
Hi, when is this likely to be resolved?

**Comment by ksfreitas** [comment]
> Hi, I'm using mysql 5.6 and it is not recognizing json fields - is mysql 5.7+ a dependency of this package?

No, you can configure using the option `documentFieldType` as `legacy`. Look my fork: https://github.com/ksfreitas/sequelize-paper-trail

**Comment by ksfreitas** [comment]
> Hi, I'm using mysql 5.6 and it is not recognizing json fields - is mysql 5.7+ a dependency of this package?

Currently you can work with any MySQL using the undocumented param `mysql` as `true`.

**Comment by erikpols** [comment]
would be nice to document the mysql parameter.
